libs/block-editor/src/lib/extensions/actions-menu.extension.ts

```diff
@@ -170,14 +171,22 @@ export const ActionsMenu = (viewContainerRef: ViewContainerRef) => {
             element: null,
             suggestion: {
                 char: '/',
-                allowSpaces: false,
+                allowSpaces: true,
                 startOfLine: true,
                 render: () => {
                     return {
                         onStart,
                         onKeyDown,
                         onExit
                     };
+                },
+                items: ({ query }) => {
```

Comment:
Can we add a comment to the docs for future reference?

docs: https://tiptap.dev/api/utilities/suggestion#items

Reply:
This

Reply:
is done